containers/ddev-webserver/files/etc/nginx/nginx-site-typo3.conf

```diff
@@ -48,6 +48,16 @@ server {
         fastcgi_param SERVER_NAME $host;
         fastcgi_param HTTPS $fcgi_https;
     }
+    
+    # Compression: Support gzipped js/css files
```

Comment:
Shouldn't this be 3 rules? One that does gzip (and adds Content-Encoding) and one each for js and css?